### DISCUSSION TITLE
Add exif tags to available Storage Template variables

### DISCUSSION THREAD

**Discussion by venyxal** [immich-app/immich#1721] [General]
More exif tags for naming would be great. Like make and camera model.

**Comment by bo0tzz** [comment]
If it's present in the exif data, that information is already included (unless I misunderstand what you mean).

![image](https://user-images.githubusercontent.com/11602424/218056668-e5aef28b-1a3d-49b5-9976-f51c928859d9.png)

**Reply by venyxal** [reply]
I mean this section in the settings panel
![Screenshot_2023-02-10-11-09-48-034-edit_com kiwibrowser browser](https://user-images.githubusercontent.com/1903739/218065266-ab7cdd19-ce3c-4bf8-aad8-aae6185b8729.jpg)

**Reply by alextran1502** [reply]
Can you help updating the discussion to make it clearer?

**Reply by Spectre5** [reply]
Sounds like he wants to get any metadata to be used as a variable.  For example, to add a variable for camera model or make.  In other words, {{camera_make}} so that he could have the images sorted by the camera used to take the image.

**Comment by Spectre5** [comment]
Another discussion I created is related to this, but is specific to just using the date/time of the image from metadata instead of the ctime on the file itself: https://github.com/immich-app/immich/discussions/1950#discussion-4929853
